prov/sockets/src/sock_comm.c

```diff
@@ -61,13 +61,15 @@
 static ssize_t sock_comm_send_socket(struct sock_conn *conn, const void *buf, size_t len)
 {
 	ssize_t ret;
-
-	ret = write(conn->sock_fd, buf, len);
+	
+	ret = send(conn->sock_fd, buf, len, 0);
```

Comment:
Do we know if send behaves differently than write for a socket?  Because write would be more efficient if not.